src/mtype.d

```diff
@@ -4763,7 +4764,10 @@ extern (C++) final class TypeSArray : TypeArray
                  * run on them for the size, since they may be forward referenced.
                  */
                 bool overflow = false;
-                if (mulu(tbn.size(loc), d2, overflow) >= 0x100_0000 || overflow) // put a 'reasonable' limit on it
+                
+                auto static_arr_size = mulu(tbn.size(loc), d2, overflow);
```

Comment:
@klickverbot This PR used to emit warning message for static arrays with more than 16MiB size (on Windows with 32bit compilation) but that's truly pointless because;
- optlink can actually handle larger static data than 16MiB
- what makes optlink crash is the total amount of static data not just one array
- as pointed out by @AndrejMitrovic there is even Unilink for linking OMF objects
- it's already possible to workaround by splitting up the array
So yes, it's unused :) Thanks for pointing out.